<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Danielas-Air.lan" failures="1" tests="3" name="Test" time="0.054" errors="0" timestamp="2022-03-16T22:05:06 CDT">
  <testcase classname="day36.Test1" name="test1" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at day36.Test1.test1(Test1.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="day36.Test1" name="test2" time="0.001"/>
  <testcase classname="day36.Test1" name="test3" time="0.002"/>
</testsuite> <!-- Test -->
